sops.yaml

```diff
@@ -1,39 +1,25 @@
 package:
   name: sops
-  version: 3.9.3
-  epoch: 1
+  version: "3.9.4"
+  epoch: 0
   description: Simple and flexible tool for managing secrets
   copyright:
     - license: MPL-2.0
 
-environment:
-  contents:
-    packages:
-      - busybox
-      - ca-certificates-bundle
-      - go
-
 pipeline:
   - uses: git-checkout
     with:
       repository: https://github.com/getsops/sops
       tag: v${{package.version}}
-      expected-commit: 3721355cbdb77d80e0686b151d8d4a60f4a56c15
-
-  # 3.9.3 update requires a tidy for some reason, we may be able to remove this in a later version.
-  # https://github.com/wolfi-dev/os/pull/35658
-  - runs: |
-      cd cmd/sops
-      go mod tidy
+      expected-commit: e794c79e2fda8b4932cf74aded946868295e291b
 
   - uses: go/build
     with:
       modroot: .
+      tidy: true
       packages: ./cmd/sops
       output: sops
 
-  - uses: strip
```

Comment:
What's the reasoning behind dropping this step?

Reply:
thanks for the reviews, it's enabled by default as part of go/build pipeline. (ref: https://github.com/chainguard-dev/melange/blob/62b32b8d4bdd1fe7b3b5c27ab40038e274d30921/pkg/build/pipelines/go/build.yaml#L58-L62)

Reply:
Thanks for the reference! From what I'm seeing, we _intentionally_ want to strip less information from Go binaries than `strip` does, right?